getting-started-guides/csp/aws/Using_EMR_Notebook.md

```diff
@@ -0,0 +1,120 @@
+# Get Started with GPU Accelerated Spark Using AWS EMR Notebook
+
+You can use the following step-by-step guide to run the example mortgage dataset using open source XGBoost library on Amazon EMR GPU clusters.  For more examples, please refer to https://github.com/rapidsai/spark-examples

```

Comment:
Can we change this line to https://github.com/NVIDIA/spark-xgboost-examples?

Reply:
Corrected now with new link, but I put spark2 as branch.  I also correct a few other links in the readme to use the realative reference.